IABot/checkIfDead.php

```diff
@@ -154,7 +135,33 @@ protected function processResult( $headers, $curlerrno, $url ) {
 		}
 	}
 
-	// Compile an array of "possible" root URLs. With subdomain, without subdomain etc.
+
+	/**
+	 * Check returned http code by making a usual GET request instead of HEAD request
+	 * @param string $url URL we are checking as a deadlink for
+	 * @return bool true if dead; false if not
+	 */
+	public function checkWithoutHeadRequest( $url ) {
+		$ch = curl_init();
+		curl_setopt( $ch, CURLOPT_URL, $url );
+		curl_setopt( $ch, CURLOPT_RETURNTRANSFER, true );
+		curl_setopt( $ch, CURLOPT_FOLLOWLOCATION, true );
+		curl_setopt( $ch, CURLOPT_AUTOREFERER, true );
+		$response = curl_exec( $ch );
+		/* Check for 404 (file not found). */
+		$httpCode = curl_getinfo( $ch, CURLINFO_HTTP_CODE );
+		if( $httpCode == 200 ) {
```

Comment:
Comparing it to line 110 on the new code, why explicitly check for 200 here?  It may cause some logic inconsistencies, will it not?